packages/docusaurus-theme-classic/src/theme/DocSidebar/index.js

```diff
@@ -121,7 +121,7 @@ function DocSidebar(props) {
 
   const {
     docsSidebars,
-    location,
+    path,
```

Comment:
Must we rename this? It'll break swizzled `DocSidebar` components. While it's ok to rename since we're in alpha, I rather we don't rename if we can.

Reply:
The reason is that now it is not a `location` object with the `pathname` property (`location.pathname`), but just a string with a path itself.

Reply:
See https://github.com/facebook/docusaurus/pull/2235/files#diff-1b5ace3ed2f9ee435090c4f4ec61c24fR111